articles/app-service-web/app-service-linux-intro.md

```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
 ms.author: naziml;wesmc
 
 ---
-# Introduction to Azure Web App on Linux
+# Introduction to Azure Web Apps on Linux
 
 [!INCLUDE [app-service-linux-preview](../../includes/app-service-linux-preview.md)]
 
 ## Overview
-Customers can use Web App on Linux to host web apps natively on Linux for supported application stacks. The following section lists the application stacks that are currently supported. 
-
-## Features
-Web App on Linux currently supports the following application stacks:
-
-* Node.js
-	* 4.4
-	* 4.5
-	* 6.2
-	* 6.6
-	* 6.9
-	* 6.10
-	* 6.11
-	* 8.0
-	* 8.1
-* PHP
-	* 5.6
-	* 7.0
-* .Net Core
-	* 1.0
-    * 1.1
-* Ruby
-	* 2.3
-
-Customers can deploy their applications by using:
+[Web Apps](https://docs.microsoft.com/en-us/azure/app-service-web/app-service-web-overview) is a fully managed compute platform that is optimized for hosting websites and web applications. Customers can use Web Apps on Linux to host web apps natively on Linux for supported application stacks. The following sections lists the application stacks that are currently supported. 
+
+### Languages
+
+|       Node.js      | PHP | .NET Core | Ruby |
+|:------------------:|:---:|:---------:|:----:|
+|      4.4, 4.5      | 5.6 |  1.0-1.1  |  2.3 |
+| 6.2, 6.6, 6.9-6.11 | 7.0 |           |      |
+|       8.0-8.1      |     |           |      |
+

```

Comment:
You could collapse this table to remove the whitespace as it isn't significant and while it does look nice from the source view, it's actually harder to update.